fromvalidation.js

```diff
@@ -0,0 +1,16 @@
+const contactMe = document.getElementById('contactme-form');
+
+function isLowerCase(str) {
+  return !/[A-Z]/.test(str);
+}
```

Comment:
Please use linter from [this](https://github.com/microverseinc/linters-config/tree/master/html-css-js) config which is for both JavaScript and Html&CSS.